x-pack/plugins/apm/server/routes/metrics/serverless/get_serverless_summary.ts

```diff
@@ -14,14 +14,67 @@ import {
   FAAS_BILLED_DURATION,
   FAAS_DURATION,
   FAAS_ID,
+  HOST_ARCHITECTURE,
   METRICSET_NAME,
   METRIC_SYSTEM_FREE_MEMORY,
   METRIC_SYSTEM_TOTAL_MEMORY,
   SERVICE_NAME,
 } from '../../../../common/elasticsearch_fieldnames';
 import { environmentQuery } from '../../../../common/utils/environment_query';
-import { calcMemoryUsedRate } from './helper';
 import { APMEventClient } from '../../../lib/helpers/create_es_client/create_apm_event_client';
+import { MetricRaw } from '../../../../typings/es_schemas/raw/metric_raw';
+import { calcEstimatedCost, calcMemoryUsedRate } from './helper';
+
+export type AwsLambdaArchitecture = 'arm' | 'x86_64';
+
+export type AWSLambdaPriceFactor = Record<AwsLambdaArchitecture, number>;
+
+async function getTransactionThroughput({
+  end,
+  environment,
+  kuery,
+  serviceName,
+  start,
+  serverlessId,
+  apmEventClient,
+}: {
+  environment: string;
+  kuery: string;
+  serviceName: string;
+  start: number;
+  end: number;
+  serverlessId?: string;
+  awsLambdaPriceFactor?: AWSLambdaPriceFactor;
+  apmEventClient: APMEventClient;
+}) {
+  const params = {
+    apm: {
+      events: [ProcessorEvent.transaction],
+    },
+    body: {
+      track_total_hits: true,
+      size: 0,
+      query: {
+        bool: {
+          filter: [
+            ...termQuery(SERVICE_NAME, serviceName),
+            ...rangeQuery(start, end),
+            ...environmentQuery(environment),
+            ...kqlQuery(kuery),
+            ...termQuery(FAAS_ID, serverlessId),
+          ],
+        },
+      },
+    },
+  };
+
+  const response = await apmEventClient.search(
+    'ger_transaction_throughout',
```

Comment:
`ger_transaction_throughout` > `get_serverless_transaction_throughout` - span names should be globally identifiable, `get_transaction_throughput` is not specific enough.